docs-web/app-protect/troubleshooting.md

```diff
@@ -77,6 +77,16 @@ Events:
 ```
 Note that in the events section, we have a `Normal` event with the `AddedOrUpdated` reason, which informs us that the configuration was successfully applied.
 
+### Replace the polcy
+
+NOTE: This mrethod only applies if using [external references](https://docs.nginx.com/nginx-app-protect/configuration/#external-references) 
+If items on the external reference change but the spec of the ApPolicy remains unchanged (even when reaplying the policy), kubernetes will not detect the update.
+In this case You can force-replace the resource. This will remove and readd the resource trigering a reload. For example:
```

Comment:
```suggestion
NOTE: This method only applies if using [external references](https://docs.nginx.com/nginx-app-protect/configuration/#external-references) 
If items on the external reference change but the spec of the ApPolicy remains unchanged (even when reapplying the policy), kubernetes will not detect the update.
In this case you can force-replace the resource. This will remove and readd the resource, triggering a reload. For example:
```